### ISSUE TITLE
Add checks for required hyperref options if it's already loaded

### ISSUE THREAD

**Issue by muzimuzhi** [ho-tex/hypdoc#5]
Starting from 083505fc0e29006900cb13989a68396337acf294 in resolving #1, `hypdoc` will only issue a warning with general message
```tex
\PackageWarning{hypdoc}{hyperref has been loaded before. Check the options!}
```
if `hyperref` is already loaded, otherwise will load `hyperref` with package-only options `hyperindex=false, pdfusetitle, pdfpagelabels`.

That means, whether `hyperref` is loaded before `hypdoc` or by `hypdoc` might produce different outputs. Since all the three options are boolean options, it seems `hypdoc` can add some checks against the underlying booleans to give a more meaningful warning message.

**Comment by nguyenvulong** [comment]
+1

**Comment by muzimuzhi** [comment]
Fixed in v1.19 (2023-10-26), see commit https://github.com/ho-tex/hypdoc/commit/440a05f2c4b9fa45558b29a146004b029a0f0b89 and CTAN release announcement https://ctan.org/ctan-ann/id/mailman.3672.1698412798.3764.ctan-ann@ctan.org.

PS: more explicit warnings were added for `hyperref` package options `hyperindex=false` and `pdfusetitle` that `hypdoc` requires, and the `pdfpagelabels` option was dropped (`hypdoc` doesn't need it starting with v1.19).
